cmd/mapt/cmd/aws/hosts/windows.go

```diff
@@ -101,6 +101,7 @@ func getWindowsCreate() *cobra.Command {
 	flagSet.Bool(spot, false, spotDesc)
 	flagSet.Bool(amiKeepCopy, false, amiKeepCopyDesc)
 	flagSet.AddFlagSet(params.GetGHActionsFlagset())
+	flagSet.AddFlagSet(params.GetCpusAndMemoryFlagset())
```

Comment:
Here paras are added but not process of the data?

Reply:
ahh yes, this is missing the corresponding code to make use of the flags, will fix